read-fonts/src/tables/variations.rs

```diff
@@ -473,6 +482,41 @@ impl<'a> DeltaRunIter<'a> {
         while self.next().is_some() {}
         self.cursor
     }
+
+    /// Skips `n` deltas without reading the actual delta values.
```

Comment:
Seems like it would be faster still to have precomputed start of y deltas. Spec opportunity that we could support with measurable perf impact?

Reply:
I imagine it would improve performance but the size increase might be substantial. I was actually thinking the opposite— that teaching write-fonts to pack the x/y streams together could shave off a nice chunk of bytes in a large CJK variable font.

Reply:
While it's not specified in the spec, I don't think *anyone* in their sane mind would get the idea that they can be encoded together.  HB doesn't support decoding them as such. I don't think FreeType does either.

Reply:
It would be an interesting data point to know what the impact on perf would be if we could jump rathern than read-to-skip.